docs/reference/components/transforms/key_value_parser.cue

```diff
@@ -3,6 +3,10 @@ package metadata
 components: transforms: key_value_parser: {
 	title: "Key Value Parser"
```

Comment:
I prefer "key-value" myself, here and in the description. Having written a book on NoSQL databases, I'm ready to pull the expert card!